aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -1080,6 +1080,43 @@ void q_batch_norm_kernel(
 
 }
 
+void fake_quantize_slice_kernel(
+    Tensor& output,
+    const Tensor& input,
+    float sc,
+    int64_t z_point,
+    int64_t quant_min,
+    int64_t quant_max) {
+  float inv_scale = 1.0f / sc;
+  auto iter = TensorIterator::unary_op(output, input);
+  cpu_kernel(iter, [&](float self) -> float {
+    return (std::fmin(
```

Comment:
David mentioned that this can be std::min instead of std::fmin as well, since you are changing the code, could you try that?